.github/workflows/build.yml

```diff
@@ -3,34 +3,34 @@ name: Build
 on: [push, pull_request]
 
 jobs:
-  openmpi:
-    runs-on: ubuntu-latest
-    timeout-minutes: 30
-    steps:
-    - uses: actions/checkout@v1
-    - name: Get OpenMPI from cache (or start caching)
-      id: openmpi-cache
-      uses: actions/cache@v1
-      with:
-        path: ./ompi216
-        key: ompi216
-    - name: download-openmpi
-      if: steps.openmpi-cache.outputs.cache-hit != 'true'
-      run: |
-        export OPENMPI_DIR=${{ github.workspace }}/ompi216
-        wget https://download.open-mpi.org/release/open-mpi/v2.1/openmpi-2.1.6.tar.gz
-        tar -xvf openmpi-2.1.6.tar.gz
-        cd openmpi-2.1.6
-        ./configure --prefix=${OPENMPI_DIR}
-        make -j
-        make install
+  # openmpi:
+  #   runs-on: ubuntu-latest
+  #   timeout-minutes: 30
+  #   steps:
+  #   - uses: actions/checkout@v1
+  #   - name: Get OpenMPI from cache (or start caching)
+  #     id: openmpi-cache
+  #     uses: actions/cache@v1
+  #     with:
+  #       path: ./ompi216
+  #       key: ompi216
+  #   - name: download-openmpi
+  #     if: steps.openmpi-cache.outputs.cache-hit != 'true'
+  #     run: |
+  #       export OPENMPI_DIR=${{ github.workspace }}/ompi216
```

Comment:
sidenote: It might be interesting to look into using the [aiida-testing](https://aiida-testing.readthedocs.io/en/latest/user_guide/mock_code.html#limitations) package. If I understand correctly, it would allow us to mock ORCA and we wouldn't have to download it here. (not in this PR of course)

Reply:
Yeah, I agree with this approach. However, it would take some time to write the tests. PR with tests would be highly appreciated.